docs/sources/dashboards/dashboard-public/index.md

```diff
@@ -52,6 +52,77 @@ publicDashboards = true
 - Click `Save Sharing Configuration` to save your changes.
 - Anyone with the link will not be able to access the dashboard publicly anymore.
 
+#### Supported Datasources
+
+Public dashboards _should_ work with any datasource that has the properties `backend` and `alerting` both set to true in it's `package.json`. However, this cannot always be
+guaranteed since plugin developers can override this functionality. Below are the datasources that are confirmed to be working with public dashboards, and the ones that should work (but have not been confirmed compatible).
+
+##### Confirmed:
+
+- Altinity plugin for ClickHouse
```

Comment:
@owensmallwood, Idea: create a three or 4-column markdown table to make use of the horizontal space on the page. You could do that for both lists. A bit less intimidating. Just a thought.

Reply:
Yeah that's a good idea. Will do that, thanks 👍